config.go

```diff
@@ -224,28 +537,76 @@ func buildTLSConfigForHttp2(config *Config) (*tls.Config, error) {
 			GetCertificate: manager.GetCertificate,
 		}
 
+	case config.LndMode == ModeRemote:
+		tlsCertPath := config.Remote.LitTLSCertPath
+		tlsKeyPath := config.Remote.LitTLSKeyPath
+
+		err := cert.GenCertPair(
```

Comment:
This appears to be generating a self-signed cert every time on startup. I'm trying to use my own cert, but it gets overwritten on launch.

Reply:
Yes, you are correct. Fixed this as well. Thank you for all the testing!